.github/workflows/run_tests.yml

```diff
@@ -55,15 +55,14 @@ jobs:
         pip install -r requirements-tests.txt
         
     - name: Run pytest with codecoverage
-      run: |
-        coverage run --source=vermouth $(which pytest) -vv vermouth --hypothesis-show-statistics
-        coverage report --omit='*/bin/pytest'
-        
+      run:  pytest --cov polyply
```

Comment:
This should be vermouth, for starters, and I would also like the --hypothesis-show-statistics flag back.

I'm kind of curious to see if that also fixes the import errors.

Reply:
sorry; early morning